### ISSUE TITLE
Training on Tiny Yolo V2

### ISSUE THREAD

**Issue by izesaon** [bastiaanv/Yolov2-tiny-tf-NCS#1]
Hi,

Thanks for the great repository. I am currently building a model to be retrained for my own dataset and wanted to clarify how I should go about retraining it using your repository if the support is provided?

**Comment by bastiaanv** [comment]
Hi,

Thank you for your interest in my repo!

As it is now, I do not support retraining in my repo using a automatic
system or something like that. You can, however, eddit the weights file to
something that would fit better on your needs.

I assume you want a system, in which you can put in a lot of images and it
would adjusts it's weights and biases. In the YoloV2 system, you have 2
systems build in: 1. Image classification 2. Object location system. I have
made, some time ago, code that would make backprop possible for the image
classification, but I have not been succesfull in creating the same for the
object location system...

So as it stands, it is not possible to retrain tge network using your own
images/dataset. It is possible to use a different weights file that fits
the networks shape and your needs

Op di 11 dec. 2018 08:42 schreef Cindy <notifications@github.com:

**Comment by izesaon** [comment]
Hi,

Does this mean that the weights I generated from retraining yolo tiny v2 on another dataset could be used for your repository?

**Comment by bastiaanv** [comment]
Yes it can! I pick the one on the yolo website, but theory ever file
generated using the yolov2 tiny network can be loaded into the repo

Op di 11 dec. 2018 16:40 schreef Cindy <notifications@github.com:

**Comment by izesaon** [comment]
Hi,

Sorry I am not too sure what you mean. But I suppose that since changing the weights is obtained when I am retraining it on another dataset, your repository would come in handy for me when I do live object detection with the stick as it can identify the objects I trained it for:) The only changes to be done would be the weights, the classes and colors right? Is there any reason for the 8 convolution layers or they are part of the Tiny Yolo V2 Network?

**Comment by izesaon** [comment]
hi, what does

load_conv_layer_bn do? is this section of code specific to the number of classes you have? because when i loaded my pretrained weights, the kernel weights generated using this function returns nan for certain values.

**Comment by izesaon** [comment]
hi, which part of the code is specific to the weights using darknet? Because when I tried to load my own weights, I get a lot of nan values for prediction but for the simo23 github repository using his code,  i didn't get any nan values. Sorry, just wanted to understand the code better as I do not understand every single part.
